src/platform_impl/macos/event.rs

```diff
@@ -86,6 +86,7 @@ pub fn get_modifierless_char(scancode: u16) -> Key {
     Key::Character(SmolStr::new(chars))
 }
 
+// Ignores all modifiers except for shift
```

Comment:
Don't think it's needed or accurate, since Alt is still applied.

Reply:
No, this `get_logical_key_char` and `charactersIgnoringModifiers` only heeds the shift key. See https://developer.apple.com/documentation/appkit/nsevent/1524605-charactersignoringmodifiers

I also verified this with `dbg!`